SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc

```diff
@@ -974,32 +978,40 @@ void HcalTB04Analysis::fillEvent(PHcalTB04Info& product) {
   product.setLongProf(eslay, eqlay);
 
   //Save Hits
-  int i, nhit = 0;
+  int nhit = 0;
+#ifdef EDM_ML_DEBUG
+  i = 0;
```

Comment:
@aandvalenzuela , shouldn't this be `int i=0;` ?

Reply:
Ah nono, I am wrong sorry. I am changing this :)